# Recall  x y z qx qy qz qw T_cam2basetrack_
translation_tracker: [-0.981056, -0.334228, -0.119872]
rotation_tracker: [0.727341, 0.216342, -0.59608, -0.262411]
